qcodes/extensions/_print_export_info.py

```diff
@@ -0,0 +1,2 @@
+def print_dataset_export_info(path: str) -> None:
```

Comment:
For testing this at a pilot station, we could consider to replace this print with a log message.